packages/@react-aria/autocomplete/src/useAutocomplete.ts

```diff
@@ -57,8 +63,8 @@ export interface AriaAutocompleteOptions<T> extends Omit<AriaAutocompleteProps<T
 }
 
 export interface AutocompleteAria<T> {
-  /** Props for the autocomplete textfield/searchfield element. These should be passed to the textfield/searchfield aria hooks respectively. */
-  textFieldProps: AriaTextFieldProps<FocusableElement>,
+  /** Props for the autocomplete input element. These should be passed to the input's aria hooks (e.g. useTextField/useSearchField/etc) respectively. */
+  inputProps: InputProps,
```

Comment:
brings up more in line with how we've named this in other hooks and narrows the type to match `FieldInputContext`.